src/OpticSim.jl

```diff
@@ -18,7 +19,8 @@ using Revise
 import GLMakie
 import Makie.AbstractPlotting
 
-unzip(a) = map(x -> getfield.(a, x), fieldnames(eltype(a)))
```

Comment:
is this ok to remove?

Reply:
not used anywhere else in the code so ok to remove.